### PR TITLE
Naming Styles Accessibility

### PR DESCRIPTION
**Pull request by dpoeschl**
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems?id=391339&_a=edit&triage=true

Ask Mode Template
=============

**Customer scenario**: A blind customer will be unable to use Naming Styles due to lack of accessibility to screen readers.

**Bugs this fixes:** https://devdiv.visualstudio.com/DevDiv/_workitems?id=391339&_a=edit&triage=true

**Workarounds, if any** None

**Risk**: Very low. Top-level feature, pretty straight-forward changes.

**Performance impact**: Essentially none.

**Is this a regression from a previous update?** No

**Root cause analysis:** This was missed during the initial feature work

**How was the bug found?** Accessibility test pass

### CODE REVIEW

**Comment by dpoeschl** [comment]
Tagging @dotnet/roslyn-ide for review

File: src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/SymbolSpecification/SymbolSpecificationDialog.xaml [hunk 1]
```diff
@@ -27,6 +27,16 @@
         <Thickness x:Key="okCancelButtonPadding">9,2,9,2</Thickness>
         <Thickness x:Key="selectDeselectButtonPadding">9,2,9,2</Thickness>
         <Thickness x:Key="textboxPadding">2</Thickness>
+        <DataTemplate x:Key="listViewDataTemplate">
+            <StackPanel Orientation="Horizontal">
+                <CheckBox x:Uid="SelectableMemberCheckBox" 
+                                          IsChecked="{Binding IsChecked, Mode=TwoWay}"
```
Comment (CyrusNajmabadi) [inline src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/SymbolSpecification/SymbolSpecificationDialog.xaml]: indentation feels weird.

Comment (dpoeschl) [inline src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/SymbolSpecification/SymbolSpecificationDialog.xaml]: That's because it is weird.

Comment (CyrusNajmabadi) [inline src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/SymbolSpecification/SymbolSpecificationDialog.xaml]: Please deweird this PR.

Comment (dpoeschl) [inline src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/SymbolSpecification/SymbolSpecificationDialog.xaml]: Deweirding complete.

**Comment by dpoeschl** [comment]
Tagging @MattGertz for ask mode consideration.

**Pull request merged**